### ISSUE TITLE
x/tools/cmd/goimports: doc lists out-of-date go-mode.el configuration

### ISSUE THREAD

**Issue by phillipao** [golang/go#24953]
https://godoc.org/golang.org/x/tools/cmd/goimports

It says to add the following to .emacs:

```emacs
(setq gofmt-command "goimports")
(add-to-list 'load-path "/home/you/somewhere/emacs/")
(require 'go-mode-autoloads)
(add-hook 'before-save-hook 'gofmt-before-save)
```

`go-mode-autoloads` is no longer used by go-mode (see dominikh/go-mode.el#222), and this line will probably produce an error. That line should be removed.

Also, I found the add-to-list line confusing, because it's not clear from context whether this is optional or required. Might be good to remove that, and then say something like "This searches load-path for goimports, so you may need to add the goimports location to load-path." (Sorry for this separate issue. Feel free to tell me to break it out to a separate issue if it's too large or controversial.)

**Comment by dominikh** [comment]
Really, the only line that is necessary is the first one. Emacs users will already have done all of the other necessary steps to get go-mode working.

**Comment by ALTree** [comment]
This was fixed in golang/tools@1c3d964395ce8f04f3b03b30aaed0b096c08c3c6, so I'm closing this issue.
